ci/common/suite.sh

```diff
@@ -11,14 +11,37 @@ FAIL_SUMMARY=""
 END_MARKER="$BUILD_DIR/.tests_finished"
 FAIL_SUMMARY_FILE="$BUILD_DIR/.test_errors"
 
+ANSI_CLEAR="\033[0K"
+
+travis_fold() {
+  local action="$1"
+  local name="$2"
+  name="$(echo -n "$name" | tr '\n\0' '--' | sed -r 's/[^A-Za-z0-9]+/-/g')"
+  name="$(echo -n "$name" | sed -r 's/-$//')"
```

Comment:
BRE syntax doesn't support `\+`.

Apple's [re_format(7)](https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man7/re_format.7.html#//apple_ref/doc/man/7/re_format) and GNU's [regex(7)](https://manpages.debian.org/jessie/manpages/regex.7.en.html) both describe BRE in similar wording:

```
Obsolete (``basic'') regular expressions differ in several respects.  `|' is an ordinary character and
there is no equivalent for its functionality.  `+' and `?' are ordinary characters, and their functionality
can be expressed using bounds (`{1,}' or `{0,1}' respectively).  Also note that `x+' in modern
REs is equivalent to `xx*'.  The delimiters for bounds are `\{' and `\}', with `{' and `}' by themselves
ordinary characters.
```